src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -595,9 +601,7 @@ CLASS zcl_abapgit_objects IMPLEMENTATION.
       RETURN.
     ENDIF.
 
-    li_obj = create_object( is_item     = is_item
-                            iv_language = zif_abapgit_definitions=>c_english ).
-
+    li_obj = create_object( is_item ).
```

Comment:
thanks for simplifying `create_object`. i had that on my todo as well.

Reply:
I'm planning some refactorings in that area too ... maybe we should synchronize the plans a bit ...